src/components/CallView/BottomBar.vue

```diff
@@ -266,7 +266,7 @@ useHotKey('f', toggleFullscreen)
 	flex-direction: row;
 	gap: var(--default-grid-baseline);
 
-	:deep(.button-vue) {
+	:deep(.button-vue:not(.button-vue--error)) {
```

Comment:
Maybe like this, to not fix it again for warning buttons, if would ever be needed?
```suggestion
	:deep(.button-vue--tertiary) {
```

Also call button should be covered  with these styles
current | expected
-- | --
<img width="173" height="55" alt="Image" src="https://github.com/user-attachments/assets/4ea0e02e-4e8f-4772-92ed-7b680574d5a2" /> | <img width="157" height="52" alt="Image" src="https://github.com/user-attachments/assets/13c0fcd1-75bf-439d-9da1-6d9c143a60ac" />